File: src/test/java/kitchenpos/application/FakeKitchenridersClient.java
package kitchenpos.application;

import kitchenpos.infra.KitchenridersClient;

import java.math.BigDecimal;
import java.util.UUID;

public class FakeKitchenridersClient implements KitchenridersClient {
    private UUID orderId;
    private BigDecimal amount;
    private String deliveryAddress;

    @Override
    public void requestDelivery(final UUID orderId, final BigDecimal amount, final String deliveryAddress) {
        this.orderId = orderId;
        this.amount = amount;
        this.deliveryAddress = deliveryAddress;
    }

    public UUID getOrderId() {
        return orderId;
    }

    public BigDecimal getAmount() {
        return amount;
    }

    public String getDeliveryAddress() {
        return deliveryAddress;
    }
}

File: src/test/kotlin/kitchenpos/application/fake/FakeKitchenridersClient.kt
package kitchenpos.application.fake

import kitchenpos.domain.KitchenridersClient
import java.math.BigDecimal
import java.util.*

class FakeKitchenridersClient : KitchenridersClient {
    override fun requestDelivery(
        orderId: UUID,
        amount: BigDecimal,
        deliveryAddress: String,
    ) {
    }
}
